dashboard/app/views/levels/_multi.html.haml

```diff
@@ -71,259 +83,21 @@
 
 = render partial: 'levels/common_audio'
 
+- # Only include multi.js once per page
+- if !@included_multi
```

Comment:
this sounds like a job for some sort of dependency requirement thing... is there a better way to do this with the code-studio package scheme? @islemaster?